### PR TITLE
2/29

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2024-02-29.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+I'm back from vacation. I know you missed my humor last week so this week I'm stepping it up. I hate the joke and I'll try and think of one for next week.
```
Comment (tbantle22) [inline emails/2024-02-29.md:3]:
```suggestion
I'm back from vacation. I know you missed my humor last week so this week I'm stepping it up. I hate the joke and I'll try to think of one for next week.
```

File: emails/2024-02-29.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### ASP.NET Core Works with Dolt
+
+We're on a mission to show that Dolt just works with all your favorite tools in all your favorite languages. This week, [the spotlight](https://www.dolthub.com/blog/2024-02-28-works-with-dolt-dotnet-webapp/) is on [.NET](https://dotnet.microsoft.com/en-us/), one of the core frameworks in Microsoft's ecosystem. 
+
+The blog comes with [sample code](https://github.com/dolthub/dolt-dotnet-webapp-sample) you can run on your own machine. You can play with a movie catalog building application complete with branches, diff, and merges.
+
+### Shallow Clone Support
+
+Databases are big and even bigger if you include all the history. What is you only want to clone the latest? In Git, there is a concept called [shallow clone](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt) and there was a [long standing feature request for Dolt to support it](https://github.com/dolthub/dolt/issues/3403).
```
Comment (fulghum) [inline emails/2024-02-29.md:23]:
```suggestion
Databases are big and even bigger if you include all the history. What if you only want to clone the latest? In Git, there is a concept called [shallow clone](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt) and there was a [long standing feature request for Dolt to support it](https://github.com/dolthub/dolt/issues/3403). 
```

**Review by fulghum — APPROVED** [review]
Looks good! 🚢

File: emails/2024-02-29.md (hunk 2 above)
Comment (tbantle22) [inline emails/2024-02-29.md:23]:
```suggestion
Databases are big and even bigger if you include all the history. What if you only want to clone the latest? In Git, there is a concept called [shallow clone](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt) and there was a [long standing feature request for Dolt to support it](https://github.com/dolthub/dolt/issues/3403). 
```

**Review by tbantle22 — APPROVED** [review]

File: emails/2024-02-29.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+### 100% `sqllogictest` Correctness
+
+We're pleased to announce that we reached [100% correctness against the `sqllogictest` suite](https://docs.dolthub.com/sql-reference/benchmarks/correctness) of 5.9M SQL queries. We started at 20% correctness, quickly got to 90% correctness, and then slowly worked our way through the 9s all the way to 100%. The last three months we got so close that we pushed for 100%.
+
+![100% Correctness](../images/correctness-percentage-chart.png)
+
+Of course, Dolt is not 100% MySQL compatible, but it almost is. We're starting to report other correctness metrics like [SQL function coverage](https://docs.dolthub.com/sql-reference/benchmarks/correctness#function-coverage), 60%, and [engine test success rate](https://docs.dolthub.com/sql-reference/benchmarks/correctness#skipped-engine-tests), 99.6%. Engine tests are our own suite of ~41,500 hand-generated tests. When we find an edge-case query where we don't return the same results as MySQL we create a skipped engine test for it. We currently have 166 skipped tests.
```
Comment (bpf120) [inline emails/2024-02-29.md:11]:
```suggestion
Of course, Dolt is not 100% MySQL compatible, but it almost is. We're starting to report other correctness metrics like [SQL function coverage](https://docs.dolthub.com/sql-reference/benchmarks/correctness#function-coverage), 60%, and [engine test success rate](https://docs.dolthub.com/sql-reference/benchmarks/correctness#skipped-engine-tests), 99.6%. Engine tests are our own suite of ~41,500 hand-generated tests. When we find an edge-case query where we don't return the same results as MySQL, we create a skipped engine test for it. We currently have 166 skipped tests.
```
